src/odemis/driver/hamamatsurx.py

```diff
@@ -807,15 +807,27 @@ def __init__(self, name, role, parent, daemon=None, **kwargs):
         parent.DevParamSet(self.location, "Delay A", 0)
         parent.DevParamSet(self.location, "Delay B", 0.00000002)
         parent.DevParamSet(self.location, "Burst Mode", "Off")
+        parent.DevParamSet(self.location, "PLL Mode", "Locked")  # Set phase locked loop mode, only used in synchroscan
```

Comment:
Don't change it by default. As it's available as a VA, if it turns out that it's convenient to have it to a specific value by default, we can set it from the microscope file.